corehq/apps/app_manager/tests/data/fixtures/expected_v2_report.xml

```diff
@@ -3,7 +3,7 @@
 	  <filters />
   </fixture>
   <fixture id="commcare-reports:c0ffee" user_id="mock-user-id" report_id="deadbeef" last_sync="2017-09-11T06:35:20" indexed="true">
```

Comment:
could remove `last_sync` from here

Reply:
alright, as long as you didn't have any other designs for it.  I'll remove.